tests/integration_tests.rs

```diff
@@ -105,3 +108,20 @@ fn run_single_test_no_exercise() {
         .assert()
         .code(1);
 }
+
+#[test]
+fn all_exercises_require_confirmation() {
```

Comment:
Checks that all exercises have the "annotation".

Not really checking the supported format exhaustively, it should at least prevent the common case where somebody completely forgets to add it in a new exercise.